iml-gui/crate/src/page/dashboard.rs

```diff
@@ -1,15 +1,14 @@
-use crate::{generated::css_classes::C, Model, Msg};
+use crate::Msg;
 use seed::{prelude::*, *};
 
-pub fn view(model: &Model) -> impl View<Msg> {
-    span![
-        pre![
-            class![C.whitespace_pre_wrap, C.text_base],
-            format!("{:#?}", model.locks),
-        ],
-        pre![
-            class![C.whitespace_pre_wrap, C.text_base],
-            format!("{:#?}", model.records)
-        ]
-    ]
+#[derive(Default)]
+pub struct Model {}
+
+pub fn view(_: &Model) -> impl View<Msg> {
+    iframe![attrs! {
+        At::Src => "https://localhost:7444/grafana/d/2vkrIIQWz/fs-dashboard?orgId=1&from=1581968869809&to=1581969769809&kiosk",
```

Comment:
```suggestion
        At::Src => "/grafana/d/2vkrIIQWz/fs-dashboard?orgId=1&from=1581968869809&to=1581969769809&kiosk",
```

We can use webpack proxy to avoid hardcoding urls.

ref:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/5a0d06eeeb6e20024d4b560af964043b4629d5ae/iml-gui/configs/webpack.config.js#L39